testsuite/forge/src/backend/local/node.rs

```diff
@@ -243,7 +246,36 @@ impl Node for LocalNode {
     }
 
     fn clear_storage(&mut self) -> Result<()> {
-        todo!()
+        // Remove all storage files (i.e., blockchain data, consensus data and state sync data)
+        let node_config = self.config();
+        let ledger_db_path = node_config.storage.dir().join(LEDGER_DB_NAME);
+        let state_db_path = node_config.storage.dir().join(STATE_MERKLE_DB_NAME);
+        let secure_storage_db_path = node_config.storage.dir().join(SECURE_STORAGE_DB_NAME);
+        let state_sync_db_path = node_config.storage.dir().join(STATE_SYNC_DB_NAME);
+
+        // Verify the files exist
+        assert!(ledger_db_path.as_path().exists() && state_db_path.as_path().exists());
+        assert!(state_sync_db_path.as_path().exists());
+        if self.config.base.role.is_validator() {
+            assert!(secure_storage_db_path.as_path().exists());
+        }
+
+        info!(
+            "Deleting ledger, state, secure and state sync db paths ({:?}, {:?}, {:?}, {:?}) for node {:?}",
+            ledger_db_path.as_path(),
+            state_db_path.as_path(),
+            secure_storage_db_path.as_path(),
+            state_sync_db_path.as_path(),
+            self.name
+        );
+        fs::remove_dir_all(ledger_db_path).unwrap();
```

Comment:
Same thing, can we `.context()?` these? That way we'll know more easily which one failed.

Reply:
Done, although, I feel so-so about this change. In tests, I find `unwrap()` and `expect` to be sufficient, i.e., the panic tells you which line failed and why. This has never not been enough. But, I've updated it.

Reply:
Oh my mistake, didn't catch that this was a test. Agreed that unwrap and expect are okay in a test.